app/views/publish/courses/_course_button_panel.erb

```diff
@@ -0,0 +1,23 @@
+<div data-qa="course__button_panel">
+  <% if course.is_published? && course.is_running? %>
+    <span class="govuk-hint govuk-!-font-size-16" data-qa="course__last_publish_date">
+      Last published <%= l(course.last_published_at&.to_date) %> at <%= course.last_published_at.strftime("%l:%M%P") %>
+    </span>
+  <% elsif course.is_withdrawn? %>
+    <span class="govuk-hint govuk-!-font-size-16" data-qa="course__withdrawn_date">
+      Withdrawn on <%= l(course.withdrawn_at&.to_date) %> at <%= course.withdrawn_at.strftime("%l:%M%P") %>
+    </span>
+  <% end %>
+
+  <div class="govuk-button-group">
+    <% if course.has_unpublished_changes? %>
+      <%= render partial: "publish/courses/course_button_panel/unpublished" %>
+    <% elsif course.is_published? && course.is_running? %>
+      <%= render partial: "publish/courses/course_button_panel/published" %>
+    <% elsif course.is_withdrawn? %>
+      <% nil %>
```

Comment:
umm this looks odd

Reply:
I was trying to work out what should take its place - there's currently no content that's meant to show up in the `withdrawn` state, really I just want something deliberately blank there, but I'm not sure what a good ruby convention is